src/runtime_src/core/tools/xbmgmt2/ReportPlatform.cpp

```diff
@@ -197,6 +197,8 @@ ReportPlatform::getPropertyTree20202( const xrt_core::device * device,
 static const std::string
 shell_status(bool shell_status, bool sc_status, int multiDSA)

```

Comment:
The variable `multiDSA `should be rename a name that represents this value.  Something similar to: `numDsaShells`

Reply:
Thanks! I couldn't think of a better variable name at the time :)